numberformat/diff.emu

```diff
@@ -185,7 +185,7 @@
           1. Let _result_ be ToRawFixed(_x_, _intlObject_.[[MinimumFractionDigits]], _intlObject_.[[MaximumFractionDigits]]<ins>, _intlObj_.[[RoundingIncrement]], _unsignedRoundingMode_</ins>).
         1. Else,
           1. <del class="block">Assert: _intlObject_.[[RoundingType]] is ~compactRounding~.</del>
-          1. <del class="block">Let _result_ be ToRawPrecision(_x_, 1, 2<ins>, _unsignedRoundingMode_</ins>).</del>
+          1. <del class="block">Let _result_ be ToRawPrecision(_x_, 1, 2).</del>
```

Comment:
No corresponding change to `proposed.emu`?

Reply:
No, because it's in a deleted section.